docs/en/connector-v2/source/Socket.md

```diff
@@ -15,58 +17,66 @@ Used to read data from Socket.
 - [ ] [parallelism](../../concept/connector-v2-features.md)
 - [ ] [support user-defined split](../../concept/connector-v2-features.md)
 
-## Options
-
-|      name      |  type   | required | default value |
-|----------------|---------|----------|---------------|
-| host           | String  | Yes      |               |
-| port           | Integer | Yes      |               |
-| common-options |         | no       | -             |
-
-### host [string]
-
-socket server host
-
-### port [integer]
+## Description
 
-socket server port
+Used to read data from Socket.
 
-### common options
+## Data Type Mapping
+
+The File does not have a specific type list, and we can indicate which SeaTunnel data type the corresponding data needs to be converted to by specifying the Schema in the config.
+
+| SeaTunnel Data type |
+|---------------------|
+| STRING              |
+| SHORT               |
+| INT                 |
+| BIGINT              |
+| BOOLEAN             |
+| DOUBLE              |
+| DECIMAL             |
+| FLOAT               |
+| DATE                |
+| TIME                |
+| TIMESTAMP           |
+| BYTES               |
+| ARRAY               |
+| MAP                 |
 
-Source plugin common parameters, please refer to [Source Common Options](common-options.md) for details
+## Options
 
-## Example
+|      name      |  type  | required | default value |                                               Description                                                |
+|----------------|--------|----------|---------------|----------------------------------------------------------------------------------------------------------|
+| host           | String | Yes      | _             | socket server host                                                                                       |
+| port           | Int    | Yes      | _             | socket server port                                                                                       |
+| common-options |        | no       | -             | Source plugin common parameters, please refer to [Source Common Options](common-options.md) for details. |
 
-simple:
-
-```hocon
-Socket {
-        host = "localhost"
-        port = 9999
-    }
-```
-
-test:
+## How to Create a Socket Data Synchronization Jobs
 
 * Configuring the SeaTunnel config file
 
-```hocon
+The following example demonstrates how to create a data synchronization job that reads data from Socket and prints it on the local client:
+
+```bash
+# Set the basic configuration of the task to be performed
 env {
   execution.parallelism = 1
-  job.mode = "STREAMING"
+  job.mode = "BATCH"
 }
```

Comment:
Add a sample title?

Reply:
Added in https://github.com/apache/seatunnel/pull/5386/files#diff-8e6fb7c11c0564bfc369d6926c2ddb6cd8d5fa04d2e617d5b34bbcde569ce268R170